client.go

```diff
@@ -1181,7 +1181,7 @@ func (f *File) WriteTo(w io.Writer) (written int64, err error) {
 	}
 
 	concurrency := int(fileSize/uint64(f.c.maxPacket) + 1) // a bad guess, but better than no guess
```

Comment:
Let’s actually use a math expression that cannot overflow, and then clamp to `f.c.maxPacket` first on that.